ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -245,7 +245,6 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
         seriesIndex++;
       }
     }
-    graphData.xAxis = xAxisData;
```

Comment:
can we just remove graphData completely?

Reply:
Dang I thought I did remove most of this in #1286, but clearly missed some places 😂